protocols/kad/CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## 0.44.0 - unreleased
```

Comment:
The way we are doing this is backwards-compatible so bumping to `0.43.2` is sufficient :)

Please also bump the version in the `Cargo.toml` to that.